src/include/log.h

```diff
@@ -238,15 +239,18 @@ struct __wt_myslot {
 #define WT_LOG_END_HEADER log->allocsize
 
 struct __wt_log {
-    uint32_t allocsize;             /* Allocation alignment size */
-    uint32_t first_record;          /* Offset of first record in file */
-    wt_off_t log_written;           /* Amount of log written this period */
-                                    /*
-                                     * Log file information
-                                     */
-    uint32_t fileid;                /* Current log file number */
-    uint32_t prep_fileid;           /* Pre-allocated file number */
-    wt_shared uint32_t tmp_fileid;  /* Temporary file number */
+    uint32_t allocsize;            /* Allocation alignment size */
+    uint32_t first_record;         /* Offset of first record in file */
+    wt_off_t log_written;          /* Amount of log written this period */
+                                   /*
+                                    * Log file information
+                                    */
+    uint32_t fileid;               /* Current log file number */
+    uint32_t prep_fileid;          /* Pre-allocated file number */
+    wt_shared uint32_t tmp_fileid; /* Temporary file number */
+#ifdef HAVE_DIAGNOSTIC
+    uint32_t min_fileid; /* Minimum file number needed */
```

Comment:
I assume this gets initialized to 0 when the struct gets initialized, presumably using something like memset?

Reply:
Yes, in `wti_logmgr_create` in `conn_log.c` the entire structure is allocated using `calloc`.